### ISSUE TITLE
Finish implementations of the training distribution bar graph

### ISSUE THREAD

**Issue by ykevu** [vanvalenlab/deepcell-label#504]
Right now, the bar graph only shows the training set values, and has a bug in the edge case where certain cell types have 0 examples in the training set, causing the graph to mislabel cell types as the plot will skip the empty cell types in the data but not the labels. The bar graph should be able to visualize the validation set distribution as well, and the code should handle the case where cell types have 0 examples.
